packages/lib-utils/src/config.ts

```diff
@@ -32,7 +39,7 @@ export const setUtilsConfig = (c: UtilsConfig) => {
  *
  * Throws an error if the reference isn't already set.
  */
-export const getUtilsConfig = () => {
+export const getUtilsConfig = (): UtilsConfig => {
   if (config === undefined) {
```

Comment:
Couldn't update this reference due to the way TypeScript guarantees that undefined was handled earlier in the flow. So I left this check in here not calling the utility I just created. 

If anyone can help me figure out that "type guard" (for the lack of a better way of saying it), let me know. 

[TS Docs on Narrowing](https://www.typescriptlang.org/docs/handbook/2/narrowing.html)